### PR TITLE
Readds some of the harder to access freon sources

### PR DESCRIPTION
**Pull request by Cyberboss**
Reverts #22025

Intentionally no CL

Why? #22029 

Multiple times since that was merged I've gotten atmos tech traitor but admons won't spawn me freon.
Only way test if the fix works is on the live servers

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
i'll spawn it in if shoot me an ahelp

**Comment by lzimann** [comment]
freon is dumb, keep it away

**Comment by XDTM** [comment]
Frozen items and tiles unfreeze with temperature now, give it a chance

**Comment by WJohn** [comment]
i hope freon is no longer buggy, broken, laggy garbage like it used to be, else i don't want anything to do with it.

**Comment by ShadowDeath6** [comment]
slip.ogg

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -272,6 +272,28 @@
...
+	required_container = /obj/item/slime_extract/darkblue
+	required_other = 1
+
+/datum/chemical_reaction/slime/slimefreeze/on_reaction(datum/reagents/holder)
+	feedback_add_details("slime_cores_used","[type]")
+	var/turf/T = get_turf(holder.my_atom)
+	T.visible_message("<span class='danger'>The slime extract begins to vibrate adorably!</span>")
+	addtimer(src, "freeze", 50, TIMER_NORMAL, holder)
+	spawn(60)
+	..()
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Wait wtf is this shit?

Comment (jughu) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Its blue slime extract silly!

Comment (Cyberboss) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: This fucking spawn though, what does it do???

Comment (optimumtact) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Nothing at all

Comment (XDTM) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: it's to not trigger the slime extract self-deletion too early, although i guess i should have used an addtimer

Comment (XDTM) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: you're right, how the fuck did that work

Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: ancient coding techniques

Comment (optimumtact) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: Are you going to remove them?

Comment (XDTM) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: sure, if you want to, i'm just very confused on why it actually delayed the ..()

Comment (optimumtact) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: So am I friend, so am I!

**Comment by Cyberboss** [comment]
This won't actually compile until #22291 is merged

**Comment by peoplearestrange** [comment]
Its an interesting new mechanic, I'd like to see people play with it a little more before its removed into non existence like so many other interesting things.

**Comment by Cyberboss** [comment]
@peoplearestrange I think it may need a slight nerf on it's specific heat, I'll be watching it carefullt

**Pull request merged**